src/registration/FormInputs.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import "./forminput.css";
+
+const FormInputs = (props) => {
+  const [focused, setFocused] = useState(false);
+  const { label, errorMessage, onChange, id, ...inputProps } = props;
+
+  const handleFocus = (e) => {
+    setFocused(true);
+  };
+
+  return (
+    <div className="formInput">
+      <label>{label}</label>
+      <input
+        {...inputProps}
```

Comment:
What are the props that are being passed to this? Add some information about Input props.